bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/model3.xml

```diff
@@ -14,6 +14,11 @@
 		<description>A Tesla Model 3 Vehicle</description>
 
 		<channels>
+			<channel id="ar_destination" typeId="ar_destination"/>
+			<channel id="ar_location" typeId="ar_location"/>
+			<channel id="ar_distancetoarrival" typeId="ar_distancetoarrival"/>
+			<channel id="ar_minutestoarrival" typeId="ar_minutestoarrival"/>
+			<channel id="ar_trafficminutesdelay" typeId="ar_trafficminutesdelay"/>
```

Comment:
It looks like the other existing channels doesn't carry such "ar" prefixes or use underscores in the naming convention for this binding. Can you rename accordingly?

Reply:
will do.